.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+name: Build Artifact
+
+on: 
+  push:
+    branches:
+      - '*'
+    # tags:        
+    #   - v*
```

Comment:
I think should be better restrict this to some branches. Maybe only `devel` for now, and maybe we can add more later.

Reply:
CI removed. Maybe can be added in another PR.